tests/integration/defs/perf/disagg/session_collect.sh

```
@@ -0,0 +1,70 @@
+    for wheel_file in $WHEEL_PATH; do
+        if [ -f "$wheel_file" ]; then
+            echo "Found wheel: $wheel_file"
+            pip3 install "$wheel_file" 2>&1 || echo "Wheel install failed, continuing..."
+            break
+        fi
+    done
+    echo "Wheel installation completed"
+
+elif [ "$INSTALL_MODE" = "source" ]; then
+    echo "Installing TensorRT-LLM from source..."
+    cd "$REPO_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)